pkg/helper/log/pipeline_reader.go

```diff
@@ -53,21 +52,22 @@ func (r *Reader) readLivePipelineLogs(pr *v1alpha1.PipelineRun) (<-chan Log, <-c
 		trC := prTracker.Monitor(r.tasks)
 
 		wg := sync.WaitGroup{}
-		taskIndex := int32(1)
+		taskIndex := 0
```

Comment:
Before this changes, initial value is 1.
and passing it by ```}(run, atomic.AddInt32(&taskIndex, 1))```

so it started from 2.
currently, it seems have no bugs, but better avoiding future confuse.